src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -357,12 +372,12 @@ private bool IsFilterCharacter(char ch)
             return GetCompletionRules().IsFilterCharacter(selectedItem, ch, filterText);
         }
 
-        private string GetCurrentFilterText(Model model, CompletionItem selectedItem)
+        private string GetCurrentFilterText(Model model, CompletionItem selectedItem, bool stopBeforeCaret = true)
         {
             var textSnapshot = this.TextView.TextSnapshot;
             var viewSpan = model.GetSubjectBufferFilterSpanInViewBuffer(selectedItem.FilterSpan);
             var filterText = model.GetCurrentTextInSnapshot(
-                viewSpan, textSnapshot, GetCaretPointInViewBuffer());
+                viewSpan, textSnapshot, endPoint: stopBeforeCaret ? (int?)GetCaretPointInViewBuffer() : null);
```

Comment:
OK, if there's no spelling mistake and you want to wrap the existing method with some other `F(` before `Min(`, the VS2013 behavior will make you disappointed.. While in VS2015, correct spelling can use `Enter`, wrap thing before can use commit character.